models/stream.go

```diff
@@ -18,12 +18,12 @@ type StreamCondition struct {
 	Operator   string                           `json:"operator,omitempty"`
 	Value      string                           `json:"value,omitempty"`
 	Type       bitmovintypes.ConditionType      `json:"type"`
-	Conditions []StreamCondition                `json:"conditions,omitempty"`
+	Conditions []*StreamCondition               `json:"conditions,omitempty"`
 }
```

Comment:
I'm a little curious why this was changed to an slice of pointers, is it something to do with the json marshalling?

Reply:
I thought it was easier to use because New*() methods are always returning pointers so you can use it as in the tests without the * operator.
```
return NewOrDisjunction(
		NewAndConjunction(
			NewAttributeCondition(bitmovintypes.ConditionAttributeFPS, "==", "25"),
			NewAttributeCondition(bitmovintypes.ConditionAttributeBitrate, "==", "14000"),
		),
		NewAndConjunction(
			NewAttributeCondition(bitmovintypes.ConditionAttributeFPS, "==", "60"),
			NewAttributeCondition(bitmovintypes.ConditionAttributeBitrate, "==", "7000"),
		),
	)
```